### PR TITLE
feat: Skip rows at the top of the file

### LINKED ISSUE #12: Skip Rows
We came into an issue where the first row of files were jibberish and needed to be skipped. I wrote a new config for this, allowing users to specify the number of lines to skip in the file. By default, the number is zero.

The draft change is at https://github.com/radbrt/tap-sftp/tree/skiprows

I might have three questions regarding this:
- Is this a common enough issue that we should add it to the repo? This isn't the first time I have had to deal with it, but maybe I have strange employers.
- Is the nomenclature "Skip rows" OK? Different tools seems to vary between specifying the number of rows to skip, and what the first row to be read should be.
- The drafts does not check if the option is set, but automatically skips a number of lines (usually 0, the default). This works, but might there be a reason to wrap it in an if-statement? I don't really see why, but who knows.

Any thoughts @pnadolny13 ?

### PR DESCRIPTION
**Pull request by radbrt**
Closes https://github.com/MeltanoLabs/tap-sftp/issues/12

Porting over the feature to skip N rows at the top of a file.

### CODE REVIEW

File: tap_sftp/sync.py [hunk 1]
```diff
@@ -64,7 +64,8 @@ def sync_file(sftp_file_spec, stream, table_spec, config, sftp_client):
             'delimiter': table_spec.get('delimiter', ','),
             'file_name': sftp_file_spec['filepath'],
             'encoding': table_spec.get('encoding', 'utf-8'),
-            'sanitize_header': table_spec.get('sanitize_header', False)}
+            'sanitize_header': table_spec.get('sanitize_header', False),
+            'clean_colnames': table_spec.get('clean_colnames', False)}
```
Comment (pnadolny13) [inline tap_sftp/sync.py]:
```suggestion
            'skip_rows': table_spec.get('skip_rows', False)}
```
@radbrt should this be skip_rows vs clean_colnames?

**Comment by pnadolny13** [comment]
@radbrt thanks for this PR! I left a comment and added a commit for updated the readme to reflect the new optional setting.

**Comment by radbrt** [comment]
Absolutely correct @pnadolny13, the copy-paste monster got me. Fixed.

**Review by pnadolny13 — APPROVED** [review]
@radbrt looks good to me! Feel free to merge when you're ready.

**Pull request merged**